Packs/Cymptom/Integrations/Cymptom/Cymptom.py

```diff
@@ -0,0 +1,250 @@
+            url_suffix="mitigations/mitigation",
+            params={"id": str(mitigation_id)},
+            timeout=timeout
+        )
+
+    def get_mitigations_subtechnique_procedure_by_id(self, sub_tech_proc_id, timeout=60):
+        return self._http_request(
+            method='GET',
+            url_suffix=f"mitigations/subtechnique-procedures/{str(sub_tech_proc_id)}",
+            timeout=timeout
+        )
+
+
+def api_test(client: Client):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to and the Connection to the service is successful.
+    :param client: Cymptom client
+    """
+    try:
+        results = client.api_test()
+        if results and results.get("status") == "ok":
+            return return_results('ok')
+        else:
+            return return_error(f"There was an error: {results.get('status', 'Failure')} - {results.get('error')}")
+    except Exception as e:
+        return_error(f"There was an error in testing connection to URL: {client._base_url},"
+                     f"API Key: {client._headers['Authorization'].split()[-1]}. "
+                     f"Please make sure that the API key is valid and has the right permissions, "
+                     f"and that the URL is in the correct form. Error: {str(e)}")
+
+
+def get_mitigations(client: Client) -> CommandResults:
+    """
+    This function uses a client argument
+    """
+    args = demisto.args()
+    timeout = args.get("timeout", 60)
+    state = args.get("state", MitigationsState.open.value)
+    timeout = int(timeout)
+
+    mitigations_results = client.get_mitigations(timeout=timeout, mitigations_state=state)
+
+    mitigations_formatted = []
+    table_headers = ["ID", "Name", "SeverityType", "AttackVectorsUsePercentage", "AttackVectorsCount",
+                     "Procedures", "Techniques", "SubTechniques", "References"]
+
+    for mitigation in mitigations_results.get("mitigations", {}):
+        extended_info = client.get_mitigation_by_id(mitigation["id"])
+        severity_type = mitigation["severity"]["name"].capitalize()
+        severity_percentage = round(extended_info["severity"]["percentage"], 2)
+
+        mitigations_formatted.append({"ID": mitigation["id"],
+                                      "Name": mitigation["name"],
+                                      "SeverityType": severity_type,
+                                      "AttackVectorsUsePercentage": severity_percentage,
+                                      "AttackVectorsCount": mitigation["vectorCount"],
+                                      "Procedures": mitigation["procedures"],
+                                      "Techniques": mitigation["mitigations"],
+                                      "SubTechniques": extended_info["subtechniques"],
+                                      "References": extended_info["references"]})
+
+    readable_output = tableToMarkdown('Mitigations', mitigations_formatted, headers=table_headers)
```

Comment:
If you want the headers to be human-readable you can pass an headerTransform function.
```suggestion
    readable_output = tableToMarkdown('Mitigations', mitigations_formatted, headers=table_headers, headerTransform=pascalToSpace)
```

Reply:
I'll keep it for next PRs (: